cfgov/v1/models/learn_page.py

```diff
@@ -373,9 +375,35 @@ class EventPage(AbstractFilterPage):
 
     template = 'events/event.html'
 
+    @property
+    def event_state(self):
+        if self.end_dt:
+            end = convert_date(self.end_dt, 'America/New_York')
+            if end < datetime.now(timezone('America/New_York')):
+                return 'past'
+
+        if self.live_stream_date:
+            start = convert_date(
+                self.live_stream_date,
+                'America/New_York'
+            )
+        else:
+            start = convert_date(self.start_dt, 'America/New_York')
+
+        if datetime.now(timezone('America/New_York')) > start:
```

Comment:
So if an event doesn't have an end date, it will stay in "present" state forever, right? Do you foresee that ever being an issue? How did we previously deal with events that had no end time?